src/chart/sunburst/SunburstPiece.ts

```diff
@@ -251,8 +252,8 @@ class SunburstPiece extends graphic.Sector {
             const rotateType = getLabelAttr(labelStateModel, 'rotate');
             let rotate = 0;
             if (rotateType === 'radial') {
-                rotate = -midAngle;
-                if (rotate < -Math.PI / 2) {
+                rotate = normalizeRadian(-midAngle);
+                if (((rotate > Math.PI / 2 && rotate <= Math.PI) || (rotate > Math.PI && rotate <= Math.PI * 1.5))) {
```

Comment:
It seems these two conditions can be merged into one condition. WDYT?

```ts
rotate > Math.PI / 2 && rotate <= Math.PI * 1.5
```

Reply:
Yeah, you are right